Added documentation explaining the requirements to make the kubeconfig generated by Okteto with the new credential plugin within the CLI (#474)
okteto/docs: src/content/reference/cli.mdx

 i  Updated kubernetes context 'cloud_okteto_com/cindylopez' in '/Users/cindylopez/.kube/config'
```

> > Since version `2.20`, the kubernetes config generated by Okteto uses a [credential plugin](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins) to get the Kubernetes credentials from your Okteto instance. This functionality requires the okteto CLI to be in your PATH. It is also required that the Okteto CLI is connected to your Okteto instace through `okteto context use <url>`

Reviewer: ```suggestion
> Starting with the Okteto CLI `2.20`, the Kubeconfig file generated by Okteto uses a [credential plugin](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins) to get the Kubernetes credentials from your Okteto instance. This functionality requires the okteto CLI to be in your PATH. It is also required that the Okteto CLI is connected to your Okteto instance through `okteto context use <url>`
```

 i  Updated kubernetes context 'cloud_okteto_com/cindylopez' in '/Users/cindylopez/.kube/config'
```

> > The kubernetes config generated uses a [credential plugin](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins) to get the Kubernetes credentials via Okteto backend that requires the okteto CLI to be in the PATH. It is also required that the Okteto CLI is connected to your Okteto instace through `okteto context use <url>`

Author: @rberrelleza Yes, that is the current behavior of the command and it won't change
